src/scripts/modules/media/body/content-baked.less

```diff
@@ -122,36 +143,83 @@ h1.example-title .text {
 
 // Helper mixin for expanding all the selectors
 .make-block(@type) {
+  #blockish>.style(@type);
```

Comment:
hmm, that's odd. Could https://github.com/Connexions/webview/blob/f7d43d9d0cc8c1d7ec76c60e2df972a5653a15e8/src/scripts/modules/media/body/content-common.less#L580-L583 be moved into both `content-baked.less` and `content-raw.less`? 

Then `content-common.less` would only define `make-block(...)` and it would only be called in either `-baked.less` or `-raw.less`?